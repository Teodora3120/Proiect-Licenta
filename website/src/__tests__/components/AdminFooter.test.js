import React from 'react'
import { shallow } from 'enzyme'
import Footer from '../../components/Footers/AdminFooter'
test('should test AdminFooter component', () => {
  const wrapper = shallow(<Footer />)
  expect(wrapper).toMatchSnapshot()
})