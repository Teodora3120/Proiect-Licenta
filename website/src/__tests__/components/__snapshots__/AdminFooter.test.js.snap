// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test AdminFooter component 1`] = `
<footer
  className="footer"
>
  <Row
    className="align-items-center justify-content-xl-between"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xl="6"
    >
      <div
        className="copyright text-center text-xl-left text-muted"
      >
        Â© 
        2023
         
        <a
          className="font-weight-bold ml-1"
          href="#"
          rel="noopener noreferrer"
          target="_blank"
        >
          Green Team
        </a>
      </div>
    </Col>
  </Row>
</footer>
`;
